chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SearchViewController.java

```diff
@@ -393,4 +400,20 @@ public void download(@RequestParam(value="edition", required=false) String editi
 		return searchMenuManager.getPopulateSearchOptions(simple);
 	}
 	
+	@ApiOperation(value="Get all developer decertifications in the CHPL", 

```

Comment:
I'm not sure why this is in the SearchViewController. Why isn't it in it's own DecertificationsController. It is a kind of search, but it's closer to a "getAll" (with a few filters) but the front end user isn't calling it with any filters at all. I could go either way on this one. @kekey1 ?